bin/kaocha

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+lein kaocha "$@"
```

Comment:
As per the docs I recommend every project to have a `bin/kaocha` wrapper, so it can be used consistently regardless of which tool you're using.